back/spec/fixtures/locales/fr-FR.yml

```diff
@@ -1,3 +1,18 @@
 fr:
   symbols:
     outside: quelque part d'autre
+  nav_bar_items:
+    home:
+      title: Accueil
+    projects:
```

Comment:
Probably you added all translations starting at this line for completeness, but they are not used. Only `home` is used in the tests. I would take out what we do not need, otherwise it may be confusing for other developers. They will see this and they will not know why it is here.